docs/mmteb/points/528.jsonl

```diff
@@ -0,0 +1,3 @@
+{"GitHub": "isaac-chung", "Review PR": 2}
+{"GitHub": "imenelydiaker", "Review PR": 2}
+{"GitHub": "KennethEnevoldsen", "Bug fixes": 1, "New dataset": 8, "Dataset annotations": 1}
```

Comment:
@imenelydiaker and @isaac-chung not really sure about the points. 

There is 4 new datasets (which are old but reformatted) (I think 1 point pr. each is reasonable)
The all of the 4 +1 have had their metadata updated (5 points)
and then a minor bug fix (1 point)

This give the same points as above, but can move them around - let me know

Reply:
This looks good :)